Week2/nadin.js

```diff
@@ -0,0 +1,181 @@
+'use strict'
+
+//strings
+
+let myString = "hello,this,is,a,difficult,to,read,sentence";
+console.log(myString);
+console.log(myString.length); 
+
+const space = " ";
+myString = 'hello' + space + 'this' + space + 'is' + space + 'a' + space + 'difficult' + space + 'to' + space + 'read' + space + 'sentence';
+console.log(myString);
+
+// Arrays
+
+let favoriteAnimals = ["blowfish", "capricorn", "giraffe"];
+let favoriteMauroAnimals = favoriteAnimals.push('turtle');
+console.log(favoriteAnimals);
+favoriteAnimals.splice(1, 0, 'meerkta');
+console.log('i think the new value of the array is:,["blowfish", "meerkta", "capricorn", "giraffe", "turtle"]');
+console.log(favoriteAnimals);
+console.log('The array has a length of:', favoriteAnimals.length);
+const index = favoriteAnimals.indexOf('meerkta');
+favoriteAnimals.splice(index, 1);
+console.log('The item I am looking for is at index:1');
+
+//1
+
+function sumThreeNumber (num1, num2, num3) {
+  return num1+num2+num3;
+}
+const total = sumThreeNumber(1,2,3);
+console.log(total);
+
+//2
+function colorCar (color) {
+return 'a' + ' ' + color + ' ' + 'car';
+}
+const colors = colorCar('red');
+console.log(colors);
+
+//3
+
+const studentObj = {
+    name :"atik",
+    course :"programing",
+    number :'20', 
+}
+
+function printProperties(obj) {
+    for (const key in obj) {
+        console.log(key, ":" , obj[key]);
+      }
+}
+printProperties(studentObj);
+
+//4
+function vehicleType(color, code) {
+   switch (code) {
+       case 1: 
+       console.log('a' + ' ' + color + ' ' +  'car');
+       break;
+       case 2: 
+       console.log('a' + ' ' + color + ' ' +  'motorbike');
+       break;
+       default:
+       console.log('parameter not supported');
+   }
+}
+
+vehicleType("blue", 2);
+
+//5
+3 === 3 ? console.log ('yes') : console.log('no');
+
+//6
+
+function vehicle(color, code, age) {
+    if (age === 5 && code === 1){
+        console.log('a' + ' ' + color + ' ' + 'used car');
+        return;
+    } else if (age === 5 && code === 2){
+        console.log('a' + ' ' + color + ' ' + 'used motorbike');
+        return;
+    }
+    switch (code) {
```

Comment:
Because you have `return` in the if statement above this switch code is never reached, you can remove the code here.

Reply:
I just want to use switch code to practice the q on another way, Thanks for the info! :)